sdks/python/README.md

```diff
@@ -2,18 +2,107 @@
 
 The Redactive Python SDK provides a robust and intuitive interface for interacting with the Redactive platform, enabling developers to seamlessly integrate powerful data redaction and anonymization capabilities into their Python applications.
 
-## Building the Python SDK
+## Installation
 
-`cd sdks/python`
+You don't need this source code unless you want to modify the package. If you just
+want to use the package, just run:
 
-`python -m pip install --upgrade build`
+```sh
+pip install --upgrade redactive
+```
 
-`python -m build`
+Install from source with:
 
-`python install -e .`
+```sh
+python -m pip install .
+```
 
```

Comment:
## Installation

In order to use the package to integrate with Redactive.ai, run:
```sh
pip install --upgrade redactive
```
There is no need to clone this repository.

If you would like to modify this package, clone the repo and install from source:
```sh
python -m pip install .
```